Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -1073,6 +1073,8 @@ WebProcessProxy& WebPageProxy::ensureRunningProcess()
 
 RefPtr<API::Navigation> WebPageProxy::loadRequest(ResourceRequest&& request, ShouldOpenExternalURLsPolicy shouldOpenExternalURLsPolicy, API::Object* userData)
 {
+	fprintf(stderr,"Access:");
+	fprintf(stderr,"loadRequest: %d",m_isClosed);
```

Comment:
Please stop using this type of logging in your commits. It's fine to have it temporarily when you debug something, but when you're done, remove it.

Also, use a \n in your logs...